examples/service-clients/azure-client/external-controller/tests/index.ts

```diff
@@ -146,15 +131,16 @@ async function setup(): Promise<void> {
 	if (createNew) {
 		window.location.hash = Date.now().toString();
 	}
-	const containerId = window.location.hash.substring(1);
+	const containerId = window.location.hash.slice(1);
 
-	const leftElement = document.getElementById("sbs-left") as HTMLDivElement;
-	if (leftElement === undefined) {
+	const leftElement = document.querySelector("#sbs-left");
+	if (!leftElement) {
```

Comment:
Good catch that this returns `null` instead of `undefined` so the original wouldn't work - however maybe prefer a strict check rather than the falsy one.

I generally prefer getElementById for efficiency but not a strong requirement.

Reply:
> I generally prefer getElementById for efficiency but not a strong requirement.

This change was to comply with our linter settings FYI. I don't have a strong preference one way or the other, but our eslint config does 🙂